.storybook/storybook.scss

```diff
@@ -1,3 +1,4 @@
+/* stylelint-disable selector-max-id */
```

Comment:
can we disable this for the declaration blocks where we know we need it vs the whole file?